cli/src/commands/create/comments.rs

```diff
@@ -279,6 +281,19 @@ fn upload_batch(
         unchanged,
     });
 
+    // Upload audio
+    for (comment_id, audio_path) in audio_paths.iter() {
```

Comment:
Nit: as this argument appears after `annotations` in the arguments list, it might be tidy if similarly this block was after the annotations block. Unless there's a logical reason not to do so?